.holo/README.md

```diff
@@ -0,0 +1,17 @@
+# hologit
+
+[hologit](https://github.com/JarvusInnovations/hologit) is a tool that facilitates manipulation of
+GitHub branches in a manner that facilitates an "obvious" GitOps workflow for CI/CD. Specifically,
+hologit allows:
+
+1. Building branches containing only a subset of repository contents (for example, a branch only including infra-related code)
+    * This action is called "projection"
+2. Bringing in contents from another repository without relying on published artifacts such as Helm charts
```

Comment:
I am a little concerned about this re: handoff. cc @themightychris 

Do we think it's appropriate or sustainable to hand off the project with this kind of dependency, particularly through an uncommon dependency mechanism (i.e., declared in places that are not necessarily standard/super visible to less experienced users)?

Reply:
Yeah but the alternatives aren't super great, they'd shift complexity to floating around in the cloud state and in people's heads rather than being baked into GitHub workflows

Reply:
I think we could at least consider referencing the published helm charts rather than bring in the jarvus cluster template. Branch projections are more "obvious" IMO and matters for the gitops workflows. I think the main concern would be upstream changes (e.g. cluster-template) sneaking into cal-itp deploys without someone realizing.